docs/openapi3.yaml

```diff
@@ -2034,3 +2040,59 @@ components:
             timeout:
               description: timout to smtp server in milliseconds
               type: number
+        benchmark_config:
+          additionalProperties: false
+          type: object
+          description: Simple mail transfer protocol.
+          required:
+          - threshold
+          - threshold_webhook_url
+          - benchmark_weights
+          properties:
+            threshold:
+              type: number
+              description: benchmark threshold
+              minimum: 0
+              maximum: 100
+            threshold_webhook_url:
+              type: string
+              format: uri
+              description: webhook url to alert when coverage less than threshold
+            benchmark_weights:
+              additionalProperties: false
```

Comment:
we have two swagger here
one is this for validations
and the other is for redocs (dev docs site)
so please update it as well